recipes/tensorstore/patches/pybind11-workspace.patch

```diff
@@ -0,0 +1,10 @@
+--- third_party/com_github_pybind_pybind11/workspace.bzl	2022-07-14 12:01:29.000000000 +0200
++++ third_party/com_github_pybind_pybind11/workspace.bzl	2022-07-14 12:01:37.000000000 +0200
+@@ -28,6 +28,7 @@
+         ],
+         sha256 = "6bd528c4dbe2276635dc787b6b1f2e5316cf6b49ee3e150264e455a0d68d19c1",
+         build_file = Label("//third_party:com_github_pybind_pybind11/bundled.BUILD.bazel"),
++        system_build_file = Label("//third_party:com_github_pybind_pybind11/bundled.BUILD.bazel"),
```

Comment:
system.build.bazel.

I might upstream this, since it's supposed to be headers-only.

Reply:
It's `bundled.BUILD.bazel` on purpose so I don't have to add a new patch. I realized that if you just use `system_build_file`, the download of the vendored version is skipped and Bazel will just find the headers elsewhere (our conda `$PREFIX`). Is my understanding correct?

Reply:
so the patch patch above (pybind11-system.patch) isn't desired?

Reply:
Oops, yep, I had to remove that one. Done :)

Once we have a new release with the patches built-in, we can point to that version and get this merged :D